pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, mkDerivation, fetchgit, cmake, libressl, SDL2, qtbase, python2 }:
+
+mkDerivation rec {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  # Submodules
+  src = fetchgit {
+    url = "https://github.com/yuzu-emu/yuzu-mainline";
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s9p5di9kcr8sh6iipv21b74c4dhn37an4khy4y7b9s9y8p1haim";
+  };
+
+  # Trick the configure system
+  preConfigure = ''
+    sed -n 's,^ *path = \(.*\),\1,p' .gitmodules | while read path; do
+      mkdir "$path/.git"
+    done
+  '';
+
+  nativeBuildInputs = [ cmake libressl ];
```

Comment:
I'm guessing that you want `libressl` in `buildInputs` instead.

Reply:
That's what i originally thought as well, however the issue described [here](https://github.com/yuzu-emu/yuzu/issues/2957) still occured when i did that. When moving it to `nativeBuildInputs` it works perfectly fine.

Reply:
Oh, OK. So do the `cmake` scripts use `libressl` or something?

Reply:
I think so, if its not already there it pulls it as a submodule, but for some reason that doesn't fix the issue so a system provided one is needed.

Reply:
This is very suspicious.

Reply:
It seems like it is built in either case.